### DISCUSSION TITLE
scheduling inside the docker image - instead of external cron?

### DISCUSSION THREAD

**Discussion by Thomas-ZA** [vogler/free-games-claimer#124] [Ideas]
Hi there,

Is there any reason the cron is exclusively called from outside the container, and that its not something that could be triggered inside the container and left running? 
would that not solve people asking about cron?
There could be an option to run once off like it does - and leave people to schedule as they want, and possibly allow people to leave the container running - and have it run a daily check if thats the case?

Just my 2c worth :)

**Comment by vogler** [comment]
Then there'll be more issues of people using an outdated image because they'd need to run e.g. [watchtower](https://github.com/containrrr/watchtower) to update it.

It's on my radar, but probably will go together with refactoring the existing scripts, adding some proper CLI or web UI.
Changing the default behavior of the image would be a breaking change, so there should be some discussion / thought behind it.

**Reply by Thomas-ZA** [reply]
like some of the guys that I see have commented - I also run portainer - pretty easy to pull new images down and refresh... could also just add a 1-liner on the readme to stop the docker, delete old container, pull new image, and start the container again... thats how I refresh portainer - just keep re-running my command from history - but it would be just as easy to copy and paste and update that way

Nifty container you have put together, definitely fixes me forgetting to add things - so much appreciated for all the effort! :)
